packages/jest-diff/src/__tests__/diff_highlight.test.js

```diff
@@ -0,0 +1,517 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+const diff = require('../');
+
+const ansiRegex = require('ansi-regex');
+const styles = require('ansi-styles');
+
+// Return diff without irrelevant escape sequences nor Expected/Received.
+const stripDiff = (a, b, options) => {
+  const result = diff(a, b, options);
+  return (
+    result &&
+    result
+      .replace(ansiRegex(), (match, offset, string) => {
+        switch (match) {
+          case styles.inverse.open:
```

Comment:
should these be in the snapshot serializer? (https://github.com/facebook/jest/blob/master/packages/pretty-format/src/plugins/convert_ansi.js)

Reply:
The goal for this subset of ConvertAnsi plugin is because the default <green> for deleted and <red> for inserted mess up the indentation, which is important in these tests.